### DISCUSSION TITLE
Commodore c128 D64 supports

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#82] [General]
Hi Pals:

If You press Shift+M over a D64 using a C128, a c64 mode is activated.

There is a way to open a d64 disk image file in c128 mode?

In the same way if You press M over a D64 disk image file, a c64 mode is activated to.

There is a way to run the first file of a d64 disk image file in c128 mode?

**Comment by frntc** [comment]
This D64 support is done via Disk2EF (easyflash cartridge with kernal hooks), this only works in C64 mode.
